build.gradle

```diff
@@ -28,6 +29,13 @@ buildscript {
         nav_version = '2.2.0-rc01'
         threetenabp_version = '1.3.0'
         timber_version = '4.7.1'
+        coroutines_android_version = '1.5.1'
+        fragment_ktx_version = '2.4.0-alpha09'
```

Comment:
Stick with a beta version, no alpha versions in commons

Reply:
got it